sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -211,53 +266,82 @@ def _to_generated(self) -> RepositoryWriteableProperties:
             can_list=self.can_list,
         )
 
-    def __getattr__(self, name):
+    def __getattr__(self, name: str) -> datetime:
```

Comment:
`__getattr__` is a generic method, not specific to a particular attribute.
```suggestion
    def __getattr__(self, name: str) -> Any:
```

Reply:
Updated